example/ck_tile/03_gemm/gemm_basic.hpp

```diff
@@ -0,0 +1,80 @@
+
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2024, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include "ck_tile/core.hpp"
+#include "ck_tile/host/kernel_launch.hpp"
+#include "ck_tile/ops/epilogue.hpp"
+#include "ck_tile/ops/gemm.hpp"
+#include <string>
+
+template <typename DataType>
+struct GemmBasicTypeConfig;
+
+template <>
+struct GemmBasicTypeConfig<ck_tile::half_t>
+{
+    using XDataType   = ck_tile::half_t;
+    using YDataType   = ck_tile::half_t;
+    using AccDataType = float;
+    using ODataType   = ck_tile::half_t; // type convert
```

Comment:
The comment at the end of line is confusing - please remove.
```suggestion
    using CDataType   = ck_tile::half_t; // type convert
```